src/eme.js

```diff
@@ -125,20 +125,43 @@ export const makeNewRequest = (player, requestOptions) => {
     keySystem
   } = requestOptions;
 
+  let timeElapsed = 0;
+  let pauseTimer;
+
+  player.on('pause', () => {
+    if (options.limitRenewalsOnPauseTime && typeof options.limitRenewalsOnPauseTime === 'number') {
```

Comment:
Do we need to document the `limitRenewalsBeforePlay` and `limitRenewalsOnPauseTime` in the README?

Reply:
Updated README in the latest commit!